models/settings.go

```diff
@@ -33,8 +33,8 @@ type MetricsResolutions struct {
 
 // SaaS contains settings related to the SaaS platform.
 type SaaS struct {
-	// Security Threat Tool enabled
-	STTEnabled bool `json:"stt_enabled"`
+	// Advisor checks disabled, false by default.
+	STTDisabled bool `json:"stt_disabled"`
```

Comment:
We can't change it like this. We store this object in PMM internal DB and once user will update his PMM advisors will be enabled automatically. But ticket says that we should enable advisors only for the fresh installations.

Reply:
I think what we need is just to change default value that we have for `STTEnabled` field.

Reply:
Yeah, Donnie suggested in the comments to enable it for all installations while keeping the same steps to disable it.

Actually it seems we can't just change default for `STTEnabled`. Problem is that we call `fillDefauts` multiple times and setting `STTEnabled` to true there will make it impossible to disable Checks. We can work around that by using `*bool` instead of `bool` but that introduces more complexity imo.

Reply:
```
UPDATE settings SET settings = settings #- '{sass, stt_enabled}';
```